axum-core/src/response/into_response.rs

```diff
@@ -478,6 +478,58 @@ macro_rules! impl_into_response {
                 res
             }
         }
+
+        #[allow(non_snake_case)]
+        impl<R, $($ty,)*> IntoResponse for (http::response::Parts, $($ty),*, R)
+        where
+            $( $ty: IntoResponseParts, )*
+            R: IntoResponse,
+        {
+            fn into_response(self) -> Response {
```

Comment:
Why is this implementation so much more complicated than the one for `(http::response::Parts, R)`? 🤔